platform_umbrella/apps/common_core/lib/common_core/installs/batteries.ex

```diff
@@ -12,7 +12,15 @@ defmodule CommonCore.Installs.Batteries do
   @aws_only_battery_types ~w(aws_load_balancer_controller karpenter)a
   @aws_battery_types ~w(battery_ca)a ++ @aws_only_battery_types
 
-  @internal_int_test_battery_types ~w(battery_core cloudnative_pg ferretdb istio_gateway metallb traditional_services)a
+  @internal_int_test_battery_types ~w(
```

Comment:
Should we make the tests install the batteries? That way test installs don't have to run so many operators?

Reply:
Yes. I've been mulling it a bit. Going to poke around and see if I can figure out something reasonably ergonomic.